### ISSUE TITLE
Design a logo.

### ISSUE THREAD

**Issue by kapoorlakshya** [maxdobeck/zeiter#12]
Design a logo (vector graphic) for Zeiter.

**Comment by kapoorlakshya** [comment]
Added a temporary logo.

**Comment by maxdobeck** [comment]
Is it attached to a local branch of yours?

**Comment by kapoorlakshya** [comment]
Yeah. It's just a placeholder. You can view it at http://zeiter.herokuapp.com

**Comment by maxdobeck** [comment]
I get a dns error. Do you have it set to auto boot when someone accesses
it?

**Comment by kapoorlakshya** [comment]
Fixed the URL. Try http://zeiter.herokuapp.com

**Comment by maxdobeck** [comment]
Damn that looks good.

**Comment by kapoorlakshya** [comment]
I like it too 😉 May tweak a few things to center align it, not sure.

**Comment by maxdobeck** [comment]
I like the left align. All the buttons can go there too. Or right align.
Either way I like the align.

**Comment by kapoorlakshya** [comment]
Sure, we can keep it left aligned. How do you feel about making it a but bigger? So it pops out a little.

**Comment by maxdobeck** [comment]
I think it's a good place holder for now. Well have to optimize for mobile
anyway. I'm more concerned with setting up roles. Not 100% sure I can just
add them to the schedules_user join table.

But I'm going to finish adding Schedules first. Make sure users have full
CRUD access.

**Comment by kapoorlakshya** [comment]
Sounds good!
